### PR TITLE
Update copy for account snaps

### PR DESCRIPTION
**Pull request by montelaidev**
## **Description**

This PR updates the copy of Account Snap related content.

## **Manual testing steps**

1. Click on Settings Icon
2. Click Experimental
3. See the Accounts Snaps section for the new copy

Part 2.
1. Click on the account picker
2. Click on `Add account or hardware wallet`
3. See the new copy for the `Add account Snap`
4. Click on `Add account Snap`
5. See the copy for the `Add Account Snap Page`

### **Before**

image
image
image
image
image
image
image
image

### **After**
Screenshot 2023-10-11 at 22 35 36
Screenshot 2023-10-11 at 22 36 53

Screenshot 2023-10-11 at 22 36 40
Updated 10/18/2023

## **Related issues**

Resolves https://github.com/MetaMask/accounts-planning/issues/103

## **Pre-merge author checklist**

- [x] I’ve followed [MetaMask Coding Standards](https://github.com/MetaMask/metamask-extension/blob/develop/.github/guidelines/CODING_GUIDELINES.md).
- [x] I've clearly explained:
  - [x] What problem this PR is solving.
  - [x] How this problem was solved.
  - [x] How reviewers can test my changes.
- [x] I’ve indicated what issue this PR is linked to: Fixes #???
- [x] I’ve included tests if applicable.
- [x] I’ve documented any added code.
- [x] I’ve applied the right labels on the PR (see [labeling guidelines](https://github.com/MetaMask/metamask-extension/blob/develop/.github/guidelines/LABELING_GUIDELINES.md)).
- [x] I’ve properly set the pull request status:
  - [x] In case it's not yet "ready for review", I've set it to "draft".
  - [ ] In case it's "ready for review", I've changed it from "draft" to "non-draft".

## **Pre-merge reviewer checklist**

- [ ] I've manually tested the PR (e.g. pull and build branch, run the app, test code being changed).
- [ ] I confirm that this PR addresses all acceptance criteria described in the ticket it closes and includes the necessary testing evidence such as recordings and or screenshots.

### CODE REVIEW

**Review by gantunesr — COMMENTED** [review]
LGTM. Not approving since some tests are failing

**Review by owencraston — DISMISSED** [review]

File: app/_locales/en/messages.json [hunk 1]
```diff
@@ -281,10 +281,10 @@
     "message": "Discover options to keep your account secure with MetaMask Snaps"
   },
   "addSnapAccountToggle": {
-    "message": "Enable \"Add Snap account\""
+    "message": "Enable \"Add Account Snap (Beta)\""
```
Comment (danroc) [inline app/_locales/en/messages.json:284]: Should be `account`

File: test/e2e/accounts/settings-add-snap-account-toggle.spec.js [hunk 2]
```diff
@@ -3,7 +3,7 @@ const { withFixtures } = require('../helpers');
 const FixtureBuilder = require('../fixture-builder');
 
 describe('Add snap account experimental settings', function () {
-  it('switch "Enable Add snap account" to on', async function () {
+  it('switch "Enable Add account snap" to on', async function () {
```
Comment (danroc) [inline test/e2e/accounts/settings-add-snap-account-toggle.spec.js:6]: `Snap`

File: ui/pages/settings/experimental-tab/__snapshots__/experimental-tab.test.js.snap [hunk 3]
```diff
@@ -206,7 +206,7 @@ exports[`ExperimentalTab with desktop enabled renders ExperimentalTab component
             <h5
               class="mm-box mm-text mm-text--body-md mm-text--font-weight-medium mm-box--margin-bottom-0 mm-box--color-text-default"
             >
-              Enable "Add Snap account"
+              Enable "Add Account Snap (Beta)"
```
Comment (danroc) [inline ui/pages/settings/experimental-tab/__snapshots__/experimental-tab.test.js.snap:209]: `account`? Is this snapshot up-to-date?

**Review by gantunesr — APPROVED** [review]

**Review by danroc — APPROVED** [review]

**Comment by plasmacorral** [comment]
Edited to update one of the after screenshots for accuracy.

**Pull request merged**